src/main/java/pl/kruczala/michal/CashMachine.java

```diff
@@ -17,16 +33,22 @@ public void setAccountBalance(int accountBalance) { //będziemy ustawiać iloś
     public int payOut(int requestedSum) throws Exception {
         if ((requestedSum <= 0)) {
             throw new ApplicationException("Requested sum is negative or equal 0 ");
-        }  if (requestedSum % 200 != 0 && requestedSum % 100 != 0 && requestedSum % 50 != 0 && requestedSum % 20 != 0) {
+        }
+        if (requestedSum % 50 != 0 && requestedSum % 20 != 0 &&
```

Comment:
ten warunek ifa jest za długi. Ja jako programista z zewnątrz muszę spędzić sporo czasu żeby zrozumieć co to właściwie sprawdza, dlatego warunki if często przenosi się do metody której nazwa mówi co to sprawdzenie robi. Drugi problem z tym warunkiem jest taki, że nie jest "globalny". W zasadzie nie wiadomo dlaczego zatrzymałeś się na 970 a nie na 1070?